### ISSUE TITLE
Missing h1 element

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#140]
The app masthead contains a `<div>` styled as an h1. This should be an `<h1>` element.

The `<h1>` and subtitle text is the same from the _Student submission_, _Student review_, and _Student submitted_ pages.
> Submit your dissertation or thesis 
> Preserve your work in the Stanford Digital Repository 

The `<h1>` and subtitle text is different for the _Reader review_ page.
> View dissertation or thesis 
> Reader review and approval
 
### Student submission page
<img width="1114" height="297" alt="Image" src="https://github.com/user-attachments/assets/829933cf-b9d3-490a-865e-f79382fb3a30" />

### Student review page
<img width="1113" height="347" alt="Image" src="https://github.com/user-attachments/assets/479064f0-99d1-4d0e-b855-34437bb39580" />

### Student submitted page
<img width="1114" height="330" alt="Image" src="https://github.com/user-attachments/assets/36a6ae33-2e09-41ab-94ba-b7caf3262f89" />

### Reader review page
<img width="1113" height="430" alt="Image" src="https://github.com/user-attachments/assets/3bd34dd4-e259-4289-89bf-dc5f3d43eb7b" />
